lib/libc/minimal/source/stdlib/calloc.c

```diff
@@ -0,0 +1,20 @@
+/*
+ * calloc.c
```

Comment:
These files should have the license boilerplate/copyright information.

Reply:
added.